cluster/gce/util.sh

```diff
@@ -3395,6 +3398,49 @@ function restart_tp_scheduler_and_controller {
   done
 }
 
+function start-mizar-scaleout {
+  echo "Installing Mizar for scale-out architecture..."
+  local -r src_dir="${KUBE_ROOT}/third_party/mizar"
+  local -r dst_dir="/tmp/mizar"
+  for (( tp_num=1; tp_num<=${SCALEOUT_TP_COUNT}; tp_num++ ))
+  do
+    mkdir -p ${dst_dir}
+    tp_kubeconfig="${RESOURCE_DIRECTORY}/kubeconfig${KUBEMARK_PREFIX}.tp-${tp_num}"
+    CLUSTER_VPC_VNI_ID="$tp_num"
+    TP_MASTER_NAME="${TPSERVER_NAME[$tp_num]}"
+    # Place mizar crds yaml
+    cp "${src_dir}/mizar-crds.yaml" ${dst_dir}
+    dst_file="${dst_dir}/mizar-crds.yaml"
+    ${KUBE_ROOT}/cluster/kubectl.sh --kubeconfig=${tp_kubeconfig} apply -f "${dst_file}"
+
+    if [[ "${tp_num}" == "1" ]]; then
+      # Place mizar daemon yaml.
+      cp "${src_dir}/mizar-daemon.yaml" "${dst_dir}"
+      dst_file="${dst_dir}/mizar-daemon.yaml"
+      sed -i -e "s@{{network_provider_version}}@${NETWORK_PROVIDER_VERSION}@g" "${dst_file}"
```

Comment:
what is the default env value for this variable. ?  NETWORK_PROVIDER_VERSION

Reply:
For now, mizar is using NETWORK_PROVIDER_VERSION="dev", we suggested mizar have a official release version after 0130 release.